README.md

```diff
@@ -5,6 +5,10 @@ This is fork from [jQuery Code Style] and follows the same licensing.
 
 This style guide should be applied to all ES6Rocks code to improve readability and consistency.
 
+## Strict mode
+
+Always use strict mode.
```

Comment:
ECMAScript modules (ES2015+) are implicitly strict. Perhaps add a note about `"use strict"` being redundant in ES2015+ module code?